tools/make/mixin.mk

```diff
@@ -0,0 +1,32 @@
+# Makefile.mixin adds mixin-specific targets.
+
+PARENT_MAKEFILE := $(firstword $(MAKEFILE_LIST))
+
+MIXIN_PATH       ?= operations/agent-flow-mixin
+MIXIN_OUT_PATH   ?= operations/agent-flow-mixin-compiled
+
+format-mixin: ## Format the mixin files.
+	@find $(MIXIN_PATH) -type f -name '*.libsonnet' | xargs jsonnetfmt -i
+
+check-mixin: ## Build, format and check the mixin files.
+check-mixin: build-mixin format-mixin check-mixin-jb check-mixin-mixtool
+	@cd $(MIXIN_PATH) && \
+	jb install && \
```

Comment:
I think you get `jb install` already from `check-mixin-jb` - if I understand this right.

Reply:
You're right, thanks!